components/ws-daemon-api/go/mock/mock.go

```diff
@@ -124,9 +124,6 @@ func (mr *MockWorkspaceContentServiceClientMockRecorder) WaitForInit(arg0, arg1
 type MockWorkspaceContentServiceServer struct {
 	ctrl     *gomock.Controller
 	recorder *MockWorkspaceContentServiceServerMockRecorder
-
-	api.UnimplementedInWorkspaceServiceServer
```

Comment:
Please rollback this change. This code needs to be added manually after the code generator